ISICbalanceExtension/TodayViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  TodayViewController.swift
+//  ISICbalanceExtension
+//
+//  Created by Rostislav Babáček on 05/05/2019.
+//  Copyright © 2019 Rostislav Babáček. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import NotificationCenter
+import SnapKit
+import ReactiveSwift
+import ACKReactiveExtensions
+
+class TodayViewController: UIViewController, NCWidgetProviding {
+    private weak var appButton: UIButton!
+    private weak var balanceTitle: UILabel!
+    private weak var balanceLabel: UILabel!
+    private let viewModel: TodayViewModeling!
+
+    // MARK: - Initialization
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        viewModel = TodayViewModel(dependencies: AppDependency.shared)
+        
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Controller lifecycle
+
+    override func loadView() {
+        super.loadView()
+
+        let balanceStack = UIStackView()
+        balanceStack.axis = .vertical
+        balanceStack.alignment = .center
+        balanceStack.spacing = 5
+        balanceStack.sizeToFit()
+        view.addSubview(balanceStack)
+
+        let balanceTitle = UILabel()
+        // we are not able to use swiftgen, because of retain cycle
+        balanceTitle.text = NSLocalizedString("balance.title", comment: "ISIC account balance")
+        self.balanceTitle = balanceTitle
+        balanceStack.addArrangedSubview(balanceTitle)
+
+        let balanceLabel = UILabel()
+        balanceLabel.font = UIFont.boldSystemFont(ofSize: 30)
+        self.balanceLabel = balanceLabel
+        balanceStack.addArrangedSubview(balanceLabel)
+
+        balanceStack.snp.makeConstraints { make in
```

Comment:
přesuň layoutování tam, kde vytváříš `balanceStack`, takhle to není moc přehledné